09-workload.md

```diff
@@ -6,37 +6,48 @@ The cluster now has an [Traefik configured with a TLS certificate](./09-secret-m
+
+   ```bash
+   kubectl apply -f ./workload/aspnetapp.yaml --context $AKS_CLUSTER_NAME_BU0001A0042_04
+   kubectl apply -f ./workload/aspnetapp-canary.yaml --context $AKS_CLUSTER_NAME_BU0001A0042_03
+   ```
+
+1. Wait until both regions are ready to process requests
 
    ```bash
    kubectl wait -n a0042 --for=condition=ready pod --selector=app.kubernetes.io/name=aspnetapp --timeout=90s --context $AKS_CLUSTER_NAME_BU0001A0042_03
+   kubectl wait -n a0042 --for=condition=ready pod --selector=app.kubernetes.io/name=aspnetapp-canary --timeout=90s --context $AKS_CLUSTER_NAME_BU0001A0042_03
+   kubectl wait -n a0042 --for=condition=ready pod --selector=app.kubernetes.io/name=aspnetapp --timeout=90s --context $AKS_CLUSTER_NAME_BU0001A0042_04
+   kubectl wait -n a0042 --for=condition=ready pod --selector=app.kubernetes.io/name=aspnetapp-canary --timeout=90s --context $AKS_CLUSTER_NAME_BU0001A0042_04
    ```
 
-1. Check your Ingress resource status as a way to confirm the AKS-managed Internal Load Balancer is functioning
+1. Check the status of your Ingress resources as a way to confirm the AKS-managed Internal Load Balancer is functioning
 
    > In this moment your Ingress Controller (Traefik) is reading your ingress resource object configuration, updating its status, and creating a router to fulfill the new exposed workloads route. Please take a look at this and notice that the address is set with the Internal Load Balancer IP from the configured subnet.
 
    ```bash
    kubectl get ingress aspnetapp-ingress -n a0042 --context $AKS_CLUSTER_NAME_BU0001A0042_03
+   kubectl get ingress aspnetapp-ingress -n a0042 --context $AKS_CLUSTER_NAME_BU0001A0042_04
    ```
 
    > At this point, the route to the workload is established, SSL offloading configured, and a network policy is in place to only allow Traefik to connect to your workload. Therefore, you should expect a `403` HTTP response if you attempt to connect to it directly.
 
-1. Deploy the same workload as another instance in the second AKS cluster
+> :book: The app team is happy to confirm to the BU0001 that their workload is now consuming less memory thanks to this update.
 
-   ```bash
-   kubectl create -f ./workload/aspnetapp.yaml --context $AKS_CLUSTER_NAME_BU0001A0042_04
-   kubectl wait -n a0042 --for=condition=ready pod --selector=app.kubernetes.io/name=aspnetapp --timeout=90s --context $AKS_CLUSTER_NAME_BU0001A0042_04
-   kubectl get ingress aspnetapp-ingress -n a0042 --context $AKS_CLUSTER_NAME_BU0001A0042_04
-   ```
+![The app team is doing some canary testing of their workload, just a few requests are routed to the new version. The ASP.NET 5 workload app is reporting less memory usage.](images/canary-testing.gif)
```

Comment:
I like it! Do we need any "now remove the 3.1 version?"  Or we okay leaving it in this state? If we leave it in this state, maybe we can at least call out that we're just leaving it like this, but of course the app team would have migrated to the new version after validation.

Reply:
at this point we have:

region 1 - ASP.NET Core 3.1 (100%)
region 2 - ASP.NET Core 3.1 (75%) - ASP.NET 5 (25%)

how do you think we should proceed from narratives perspective. I was thinking to keep this "as-is"

Reply:
Just created a AzDO item to follow up on this, so we can keep discussion around.